src/bio/bio_context.c

```diff
@@ -408,7 +408,7 @@ int bio_mc_destroy(struct bio_xs_context *xs_ctxt, uuid_t pool_id, enum bio_mc_f
 		if (rc == -DER_NONEXIST) {
 			return 0;
 		} else if (rc) {
-			D_ERROR("Qeury data blob for pool "DF_UUID" tgt:%u failed. "DF_RC"\n",
+			D_ERROR("Query data blob for pool " DF_UUID " tgt:%u failed. " DF_RC "\n",
 				DP_UUID(pool_id), xs_ctxt->bxc_tgt_id, DP_RC(rc));
```

Comment:
This would be a good opportunity to update these logging lines to the new standard if you're updating them anyway - one less thing to come back and do later.  Not a big issue though.
```suggestion
			DL_ERROR(rc, "Query data blob for pool " DF_UUID " tgt:%u failed",
				DP_UUID(pool_id), xs_ctxt->bxc_tgt_id);
```

Reply:
If I need to push again I will